mne/channels/tests/test_montage.py

```diff
@@ -252,13 +256,8 @@ def test_montage():
     # Bvef is either auto or mm in terms of "units"
     with pytest.raises(ValueError, match='be "auto" or "mm" for .bvef files.'):
         bvef_file = op.join(tempdir, 'test.' + 'bvef')
-        read_montage(bvef_file, unit='m')
-
-    # Test reading in different letter case.
-    ch_names = ["F3", "FZ", "F4", "FC3", "FCz", "FC4", "C3", "CZ", "C4", "CP3",
-                "CPZ", "CP4", "P3", "PZ", "P4", "O1", "OZ", "O2"]
-    montage = read_montage('standard_1020', ch_names=ch_names)
-    assert_array_equal(ch_names, montage.ch_names)
+        with pytest.deprecated_call():
+            read_montage(bvef_file, unit='m')
```

Comment:
read_dig_captrack ? Also look at this https://github.com/mne-tools/mne-python/blob/master//mne/channels/_dig_montage_utils.py#L189 

the location of reference channel is present like for a .fif file.

Reply:
good catch. https://github.com/mne-tools/mne-python/issues/6788

Reply:
good catch. I've open an issue https://github.com/mne-tools/mne-python/issues/6788